lib/nerves_hub_link/extensions/health/default_report.ex

```diff
@@ -59,6 +61,25 @@ defmodule NervesHubLink.Extensions.Health.DefaultReport do
     |> Enum.reduce(%{}, &Map.merge/2)
   end
 
+  defp get_metric_sets() do
+    metric_sets =
+      Application.get_env(:nerves_hub_link, :health, [])
+      |> Keyword.get(:metric_sets, @default_metric_sets)
+      |> Enum.map(fn metric_set ->
+        if metric_set in [:default, :defaults] do
+          @default_metric_sets
+        else
+          metric_set
+        end
+      end)
+      |> List.flatten()
+
+    for metric_set <- metric_sets,
+        {k, v} <- metric_set.sample(),
+        into: %{},
+        do: {normalize_key(k), v}
```

Comment:
normalize_key only checks that it is printable and then inspects it. Does that mean atoms would just be converted to `":some_atom"` via inspect? If so, that would leave the same problem of duplicated data with key variations

Reply:
There is a version of `normalize_key` that checks if it is not binary and then will use `to_string`

https://github.com/nerves-hub/nerves_hub_link/pull/279/files#diff-b1751145aa12511d57bcf440036ff0cfea3945870a4048019fecd12fd91d0e78R123-R126